releaseNotes/JavaScriptProvider.md

```diff
@@ -2,6 +2,13 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0-preview.3 - March 21, 2023
+## Enhancements
+- Added support for updating configuration on demand without causing an application to restart. Dynamic refresh can be enabled by passing `RefreshOptions` when loading the configuration. A new API, `refresh`, has been added to trigger the refresh operation. When this API is called, it checks the server for any new changes to the `watchedSettings` after a specified time interval, `refreshIntervalInMs`. Once there is a change detected on a watched setting, all loaded settings will be updated. See usage in [this example](https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/refresh.mjs). [#21](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/21)
+- Added support for consuming settings as an object. A new API, `constructConfigurationObject`, has been added to construct an object based on the hierarchical keys of the loaded settings. It helps to minimize code changes when switching from local configuration to Azure App Configuration. [#49](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/49)
+- Added error handling delay for `load` method. The `load` method is usually called in the application's startup code path. Unhandled exceptions in the `load` method can cause the application to crash, resulting in crash loops as orchestrators attempt to restart the application. To mitigate back-to-back crash loops and prevent overloading the server with requests, a minimum time is waited to propagate fatal errors. [#35](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/35)

```

Comment:
Not really a customer facing feature. I would skip it from the release notes.

Reply:
removed in f4af1bf